src/rust/ide/view/graph-editor/src/lib.rs

```diff
@@ -2475,13 +2449,15 @@ fn new_graph_editor(app:&Application) -> GraphEditor {
     cursor_style_on_edge_drag_stop <- out.all_edges_attached.constant(default());
     cursor_style_edge_drag         <- any (cursor_style_edge_drag,cursor_style_on_edge_drag_stop);
 
+    breadcrumb_style <- model.breadcrumbs.frp.outputs.pointer_style.map(|s| s.clone());
```

Comment:
if you really need to create such a shorter name, its always better to be explicit about it:
```rust
let breadcrumb_style = model.breadcrumbs.frp.outputs.pointer_style.clone_ref()
```

HOWEVER, as my comment above suggests, it wouldb e good if you use `define_endpoints!` to define fpr nets in breadcrumbs (and breadcrumbs should deref to frp), so this line would be simplified:

```rust
let breadcrumb_style = model.breadcrumbs.pointer_style.clone_ref()
```